homeassistant/components/climate/radiotherm.py

```diff
@@ -110,6 +145,23 @@ def device_state_attributes(self):
         }
 
     @property
+    def fan_list(self):
+        """List of available fan modes."""
+        return self._fan_list
+
+    @property
+    def current_fan_mode(self):
+        """Return whether the fan is on."""
+        return self._fmode
+
+    def set_fan_mode(self, fan):
+        """Turn fan on/off."""
+        if fan == STATE_AUTO or fan == STATE_OFF:
```

Comment:
Why do you list `STATE_AUTO` as a fan mode if it does the same as `STATE_OFF`? Or am I missing something?